<launch>
	
	<!-- load obstacle map configurations from YAML file to parameter server -->
	<rosparam file="$(find terrain_server)/config/obstacle_map.yaml" command="load"/>
	
	<node pkg="terrain_server" type="obstacle_map_server" name="terrain_obstacle_map_server" output="screen">
		<remap from="obstacle_map" to="/obstacle_map" />
		<param name="world_frame" type="string" value="odom" />
		<!-- Base frame of the robot -->
		<param name="base_frame" type="string" value="base_link" />
	</node>
	
</launch>